Add snippets for `sorry`, comments, docstrings, and declarations

jesse-michael-han opened the pull request:
Add snippet for license boilerplate

Use mirror in license-boilerplate

Add def-namespace and inductive-namespace

Add .gitignore

cipher1024 commented on snippets/lean-mode/license-boilerplate:
```diff
@@ -0,0 +1,11 @@
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): $1 $2
+
+$3
```
Should we have a default string like `put description here` for `$3`?

cipher1024 commented on snippets/lean-mode/have:
```diff
@@ -0,0 +1,6 @@
+# key: have
+# --
+have ${1:this} : $2,
+  by {$3},
```
can we put `by` as part of the default value in this and the other snippets?

cipher1024 commented on snippets/lean-mode/let:
```diff
@@ -3,4 +3,5 @@
 # --
 let ${1:this} : $2,
-  by {$3},
+  ${3:by {$4}}
```
for this one, we could default to `:=` and put it on the previous line

jesse-michael-han replied:
done; I also removed the type ascription since it's not common to use it with `let`

Commit:
Add snippets for `sorry`, comments, docstrings, and declarations

Add snippet for license boilerplate

Use mirror in license-boilerplate

Add def-namespace and inductive-namespace

Add .gitignore

Commit:
Move files to correct place

Commit:
Add "put description here" to license snippet

Commit:
Snippets for have, let, show, and suffices

Commit:
Move `by` into the default value for have, let, show, suffices

Commit:
Put `:=` as default for let, remove type ascription

Commit:
Add commas

Commit:
Add snippet for `by`
